addusers.py

```diff
@@ -378,7 +430,9 @@ def parse_rows(rows, select_user=None):
     rows = sheet.get_all_rows("Form Responses 1")
 
     try:
-        content, bad_rows = parse_rows(rows, select_user=args.user)
+        content, bad_rows = parse_rows(rows, select_user=args.user,
```

Comment:
connecting to Google is typically the slowest part of this script so we always want to avoid doing it unnecessarily